import-service/src/functions/importFileParser/index.ts

```diff
@@ -0,0 +1,15 @@
+import { handlerPath } from '@libs/handler-resolver';
+
+export default {
+  handler: `${handlerPath(__dirname)}/handler.main`,
+  events: {
```

Comment:
If it works, great, but I believe that events should be an array
events: [{s3:...}]